meta/FlexibleSUSY.m

```diff
@@ -1986,18 +1986,65 @@ corresponding tadpole is real or imaginary (only in models with CP
             partialWidthCalculationPrototypes = "", partialWidthCalculationFunctions = "",
             calcAmplitudeSpecializationDecls = "", calcAmplitudeSpecializationDefs = "",
             partialWidthSpecializationDecls = "", partialWidthSpecializationDefs = "",
-            smParticleAliases, solverIncludes = "", solver = ""},
+            smParticleAliases, solverIncludes = "", solver = "", contentOfPath = $Path, modelName},
+
+            modelName =
+               If[SARAH`submodeldir =!= False,
+                  SARAH`modelDir <> "/" <> SARAH`submodeldir,
```

Comment:
If it is a `modelName`, then shouldn't it be via `-`, like `Leptoquarks-ScalarLeptoquarks.m`?